protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/deployment/DeploymentRecord.java

```diff
@@ -27,8 +29,17 @@
   private final ArrayProperty<ProcessMetadata> processesMetadataProp =
       new ArrayProperty<>(PROCESSES, new ProcessMetadata());
 
+  private final ArrayProperty<DecisionRecord> decisionMetadataProp =
+      new ArrayProperty<>("decisionsMetadata", new DecisionRecord());
+
+  private final ArrayProperty<DecisionRequirementsMetadataRecord> decisionRequirementsMetadataProp =
+      new ArrayProperty<>("decisionRequirementsMetadata", new DecisionRequirementsMetadataRecord());
```

Comment:
🔧 For consistency in this class, it would be nice to extract `"decisionsMetadata"` and `"decisionsRequirementsMetadata` as a constants, similar to `RESOURCES` and `PROCESSES`.

Reply:
Yes, the names of the other properties are extracted as constants. But these constants are not reused somewhere else. So, I don't see a value in creating more constants in this record.